### DISCUSSION TITLE
Procedure di Scrutinio

### DISCUSSION THREAD

**Discussion by Lucaccc** [iisgiua/giuaschool#440] [Richieste di aiuto] [answered]
Buonasera, volevo un'informazione relativa agli scrutini.
Nella sezione di configurazione è presente "Esami Giudizio Sospeso": questa voce fa riferimento, quindi, solo allo scrutinio finale, è corretto?

Come si può fare per gestire gli esami di recupero relativi, invece, al trimestre?

**Comment by trinko** [comment, marked as answer]
Non esistono esami per il recupero dei debiti per i periodi intermedi. 
Si svolgono normali verifiche.

**Reply by Lucaccc** [reply]
> Non esistono esami per il recupero dei debiti per i periodi intermedi. Si svolgono normali verifiche.

Nella mia scuola non funziona così...ad ogni modo grazie, troveremo un altro modo per la comunicazione dei voti.
